custom/icds_reports/reports/prevalence_of_stunting.py

```diff
@@ -33,7 +33,7 @@ def get_data_for(filters):
         if not show_test:
             queryset = apply_exclude(domain, queryset)
         if 'age_tranche' not in config:
-            queryset = queryset.exclude(age_tranche=72)
+            queryset = queryset.filter(age_tranche__lt=72)
```

Comment:
hey, @sudh-kvantum,  I meant in all places were you have change age_tranche__lt=72. should be changed to filter instead of exclude. So that its correct in all places.

Reply:
Done, please review. Thanks!